static/js/util/profile_edit.js

```diff
@@ -19,6 +20,49 @@ import {
 // in to whatever component it is used in.
 
 /**
+ * bind this to this.boundRadioGroupField in the constructor of a form component
+ * to update radio buttons.
+ * pass in the name (used as placeholder), key for profile, and the options.
+ *
+ * @param keySet {String[]} Path to the field
+ * @param label {String} Label for the field
+ * @param options {Object[]} A list of options for the select field
+ * @returns {ReactElement}
+ */
+export function boundRadioGroupField(keySet, label, options) {
+  const {
+    profile,
+    updateProfile
+  } = this.props;
+
+  let onChange = e => {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, e.target.value);
+    updateProfile(clone);
+  };
+
+  const selectedValue = _.get(profile, keySet);
+  const radioButtons = _.map(options, (obj) => {
+    let label = (<p key={obj.value}
+      className="radio-label">{obj.label}
+      <span className="radio-label-hint"> - {obj.helper}</span>
+    </p>);
+
+    return (<RadioButton
+      value={obj.value}
+      label={label}/>);
+  });
+   
+  return (<RadioButtonGroup
+    name={label}
+    onChange={onChange}
+    defaultSelected={selectedValue}>
+      {radioButtons}
+    </RadioButtonGroup>
```

Comment:
when returning JSX I think we should use one of these two styles:

```js
return <div>
  { foo }
</div>;

// or

return (
  <div>
    { foo }
  </div>
);
```

I prefer the latter b/c then the opening and closing tags line up, but it's a style preference which one you want to use.